Assets/Tests/InputSystem.Editor/XRIPackageTest.cs

```diff
@@ -0,0 +1,82 @@
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEditor;
+using UnityEngine.TestTools;
+using UnityEditor.PackageManager;
+using UnityEditor.PackageManager.Requests;
+
+// Disable irrelevant warning about there not being underscores in method names.
+#pragma warning disable CA1707
+
+public class XRIPackageTests
+{
+    static AddRequest XRAddRequest;
+    static RemoveRequest XRRemoveRequest;
+
+    /// <summary>
+    /// TearDown removes the added XRI package again.
+    /// If you are adding a 2nd test that needs the XRI package adjust this.
```

Comment:
As far as a remember unity. [UnityTearDown]/[UnityTearDown] are invoked before/after each test. So when  AdddingLatestXRIPackageThrowsNoError ends `com.unity.xr.interaction.toolkit` will be uninstalled. So comment should be adjusted.